src/Service.Host/client/src/css/landscapePrint.css

```diff
@@ -0,0 +1,16 @@
+@media print {
+    @page {
+        size: 297mm 210mm ; 
+        
+        margin: 0mm;
+        margin-right: 0mm; 
+      }
+}
+
+.date-for-printing {
+    display: none;
```

Comment:
Does the date show in the print preview? Could be wrong, or maybe it uses the shared css and this just overrides the sizes, but I thought it'd need a display: block inside the @media print bit too. Probably doesn't matter if I'm hopefully going to put it into the shared css in a mo anyway

Reply:
Ah it'll use the shared one yeah! Ignore that ^

Reply:
yes. it shows when printing. it still picks that up from the shared @media print. I've only overridden the size